src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -147,6 +147,13 @@ You can opt out of Android's Activity Instrumentation and App Start Instrumentat
 </manifest>
 ```
 
+### Slow and Frozen Frames
+
+Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the transactions you capture.
```

Comment:
```suggestion
Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the [Android's Activity](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) transactions.
```